src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -255,19 +255,12 @@ class ExternalIssueActions extends AsyncComponent {
               </li>
             </ul>
             <Modal.Body>
-              {action === 'create' && (
+              {action && (
                 <ExternalIssueForm
+                  key={action}
```

Comment:
hmmm we should add a comment here because, guaranteed, someone will read this and think that `key` isn't required.